integration-tests/src/test/java/com/redhat/service/bridge/integration/tests/steps/BridgeSteps.java

```diff
@@ -61,7 +63,21 @@ public void createNewBridge(String testBridgeName) {
         context.newBridge(testBridgeName, response.getId(), systemBridgeName);
     }
 
-    @And("^the list of Bridge instances is containing the Bridge \"([^\"]*)\"+$")
+    @Given("^create a new Bridge \"([^\"]*)\" is failing with HTTP response code (\\d+)$")
+    public void createNewBridgeIsFailingWithHTTPResponseCode(String testBridgeName, int responseCode) {
+        BridgeResource.addBridgeResponse(context.getManagerToken(), testBridgeName)
+                .then()
+                .statusCode(responseCode);
+    }
+
+    @When("^create a fake Bridge \"([^\"]*)\"$")
+    public void addFakeProcessorToBridge(String testBridgeName) {
```

Comment:
the method name doesn't seem to correspond with a step description

Reply:
from another PR but will correct it